content/blog/getting-started-with-the-hpe-greenlake-developer-portal.md

```diff
@@ -0,0 +1,88 @@
+
+![](/img/blog-greenlake-dev-portal1.png)
+
+## Guides: Learn how to get started using the HPE GreenLake Developer Portal. 
+
+ The minimal requirement of being able to start using the portal is to have an HPE user account or an HPE GreenLake Account. To leverage the HPE GreenLake platform APIs, the user will need to create an API client credentials for the HPE GreenLake platform. With the API client credentials, the user will be able to create an access token.  
+
+One cannot generate an access token without having an HPE account part of a workspace. You will be guided through all the different steps to create the account either through the present portal or through the HPE GreenLake portal. 
+
+It is important to note that some content on the portal is private or only visible to a restricted group (for example, partners or developers). The portal comes with the following roles:  
+
+ *Public* 
+
+Guest: Every visitor will have this role with a single permission to read all public content on the portal. 
+
+Authenticated-User: Every logged in user will have this role. 
+
+*Partner* 
+
+Users with the Partner role can access restricted content that is specific to a Partner.  
+
+## Services:  Discover the HPE Greenlake platform APIs  
+
+When clicking on the second tile (Services), provides you with access to the HPE GreenLake APIs. But, as mentioned previously, an API needs a token to be tested. Therefore, the very first step described in the documentation n API [here](https://developer.greenlake.hpe.com/docs/greenlake/guides/#generate-or-reset-application-credentials) is to create API client credentials for the HPE GreenLake platform and generate an access token. 
+
+![](/img/blog-greenlake-dev-portal2.png)
+
+Once you have your token, you can try the different APIs that are related to the different services listed on the left-hand side of the page.  
+
+[The HPE GreenLake audit log service](https://developer.greenlake.hpe.com/docs/greenlake/services/audit-logs/public/), for instance, will offer a collection of RESTful APIs for publishing audit logs, managing configurations, and retrieving application-specific and overall platform logs.
+
+Each service section offers, at a minimum, an overview, a guide, the Open API specification, and the associated API reference documentation. Some services provide additional parts to dive into more specific's areas of the service. For instance, HPE GreenLake for Compute Ops Management service provide additional details about event webhooks and jobs. A webhook is an HTTP-based callback function that allows lightweight, event-driven communication between two APIs.  A job is a multi-step task that is managed by Compute Ops Management to perform an action on a resource. For example, performing power operations or firmware updates on a server. 
```

Comment:
Change "....dive into more specific's areas..." to "...dive into more specific areas..."
In the next sentence, modify it to read: "For instance **the** HPE GreenLake for Compute Ops Management service provid**es** additional..."